docs/hardware.rst

```diff
@@ -408,13 +408,26 @@ To assist you, we have compiled the following partial list of
-| Model                   | Testing Date   | Tails Versions   | Price (new)        | Price (used)       | Notes                                                                                                                                                       |
-+=========================+================+==================+====================+====================+=============================================================================================================================================================+
-| HP LaserJet 400 M401n   | 06/2015        | 1.4              | $178.60 (Amazon)   | $115.00 (Amazon)   | Monochrome laser printer. Heavy (10 lbs.) When adding the printer in Tails, you need to set "Make and model" to "HP LaserJet 400 CUPS+Gutenprint v5.2.9".   |
-+-------------------------+----------------+------------------+--------------------+--------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| HP Deskjet 6940         | 04/2015        | 1.3.2            | $639.99 (Amazon)   | $196.99 (Amazon)   | Monochrome Inkjet printer                                                                                                                                   |
-+-------------------------+----------------+------------------+--------------------+--------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------+
++-------------------------+--------------+----------------+--------------------+
+| Printer Model           | Testing Date | Tails Versions | Printer Type       |
++=========================+==============+================+====================+
+| HP Deskjet F4200        | 06/2017      | 3.0            | Color Inkjet       |
++-------------------------+--------------+----------------+--------------------+
+| HP Deskjet 1112         | 06/2017      | 3.0            | Color Inkjet       |
++-------------------------+--------------+----------------+--------------------+
+| HP LaserJet 400 M401n   | 06/2015      | 1.4            | Monochrome Laser   |
++-------------------------+--------------+----------------+--------------------+
+| HP Deskjet 6940         | 04/2015      | 1.3.2          | Monochrome Injket  |
++-------------------------+--------------+----------------+--------------------+
+
+.. note:: We've documented both the HP Deskjet F4200 and HP LaserJet 400 M401n
+         with screenshots of the installation process, in our section on
+         `setting up printers in Tails
+         <https://docs.securedrop.org/en/stable/tails_printing_guide.html>`_.
+         While the F4200 installed automatically, the 400 M401n required that we
+         set "Make and model" to "HP LaserJet 400 CUPS+Gutenprint v5.2.9" when
+         manually configuring the
+         drivers.
```

Comment:
Two style notes:

1. The line wrapping on the last line is funky; `drivers.` should not be on its own line.
2. I recommend indenting lines after the first line in a directive to the same level as the marker, which is 3 spaces in this case (the marker is `.. `). I know we don't follow this style convention throughout the docs, but I'd like to change that (after 0.4, of course). The reStucturedText [docs on indentation](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#indentation) say it is a requirement:

    > Several constructs begin with a marker, and the body of the construct must be indented relative to the marker.